lib/Cargo.toml

```diff
@@ -6,7 +6,7 @@ license = "MIT OR Apache-2.0"
 name = "ostree-ext"
 readme = "README.md"
 repository = "https://github.com/ostreedev/ostree-rs-ext"
-version = "0.10.7"
+version = "0.11.0"
```

Comment:
I think you should `git fetch origin && git rebase origin/main` since this commit is already in main.